airbyte-webapp/src/components/JobItem/components/MainInfo.module.scss

```diff
@@ -8,12 +8,14 @@
   .titleCell {
     display: flex;
     color: colors.$dark-blue;
+    min-width: 0;
 
     .statusIcon {
       display: flex;
       align-items: center;
       justify-content: center;
       width: 75px;
+      min-width: 75px;
```

Comment:
Flexbox handles min-width in an odd way.  This prevents it from resizing the `statusIcon` and messing up the flow when the content of the title cell is very long.

Reply:
Found this helpful if anyone is curious on why this is needed: https://stackoverflow.com/questions/36230944/prevent-flex-items-from-overflowing-a-container